apps/omg_child_chain/config/config.exs

```diff
@@ -9,4 +9,9 @@ config :omg_child_chain,
   fee_specs_file_name: "fee_specs.json",
   ignore_fees: false
 
+config :omg_child_chain, OMG.Utils.Tracer,
```

Comment:
We're providing a separate config for each app because the `service` name is different. This will be passed to Datadog and will help us see what part of the app is sending the trace.